src/scss/components/_media-element.scss

```
@@ -1,3 +1,7 @@
+@use '@justeat/pie-design-tokens/dist/jet'; as dt;
```

I'm guessing the first semi-colon shouldn't be there?

```suggestion
@use '@justeat/pie-design-tokens/dist/jet' as dt;
```

---

🤦